R/BuildModel.R

```diff
@@ -1,5 +1,6 @@
 # Define model version
-model_ver <- utils::packageDescription(pkg = "stateior", fields = "Version")
+# model_ver <- utils::packageDescription(pkg = "stateior", fields = "Version")
+model_ver <- NULL
 
```

Comment:
@bl-young @jvendries I realize `model_ver` was created to assist downloading specific versions of data from Data Commons, and now it just makes things easier for us to disable it and force all functions to use the latest raw data from DC.

Reply:
I think this makes sense to me. Ultimately then verison number only matters when reading stateio files from useeior?

Reply:
Yes, it should only matter when being used in useeior.